CHANGELOG.md

```diff
@@ -158,6 +158,22 @@ _Infrastructure:_
 - JavaThemis for Android and desktop Java is now published in the Maven Central repository ([#786](https://github.com/cossacklabs/themis/pull/786), [#788](https://github.com/cossacklabs/themis/pull/788)).
 - MSYS2 builds for Windows are now checked by CI ([#791](https://github.com/cossacklabs/themis/pull/791)).
 
+## [0.13.8](https://github.com/cossacklabs/themis/releases/tag/0.13.8), April 30th 2021
+
+**Hotfix for Apple platforms:**
+
+- Updated OpenSSL to the latest 1.1.1k for SMP and attached `themis.xcframework`. (iOS and macOS).
+- New Swift and Objective-C example projects: SPM for iOS and macOS.
```

Comment:
SMP is symmetric multiprocessing. SPM is Swift Package Manager. You want the latter.

Ditto below.

Though, I don't think we need to mention attaching `themis.xcframework` at all, neither here in highlights, nor in platform-specific details. Attaching a new binary is now a standard procedure for SPM. SPM support has been announced in 0.13.7, from now on users can rely on this support until further notice.

Reply:
This line tells that new OpenSSL version is available with SPM (using XFC) or by just adding XCF to the project without SPM. It is important because OpenSSL 1.1.1k is still not available for Themis via CocoaPods and Carthage (it will be our next step). It is not about announcing SPM in general.